packages/devextreme-cli/src/templates-ts/react/application/src/app-navigation.tsx

```diff
@@ -0,0 +1,21 @@
+export const navigation = [<%=^empty%>
```

Comment:
Maybe we need to add a navigation type?

Reply:
The typescript does not underline with an error the navigation. Should we add type where typescript understand this type?